can/bit_timing.py

```diff
@@ -24,10 +24,18 @@ class BitTiming:
     """
 
     sync_seg = 1
+    # generator that produces all possible combinations of brp, sjw, tseg1 and tseg2
+    _calc = [
```

Comment:
Actually this is list with 32768 entries.

Reply:
sorry forgot to change the []'s to ()'s whooops